### ISSUE TITLE
Add the eyes-only-user-access shortcode

### ISSUE THREAD

**Issue by stevejburge** [publishpress/publishpress-permissions#148]
It would be very useful for users if we included the eyes-only-user-access shortcode feature
<img width="544" alt="Screen Shot 2020-08-10 at 3 26 47 PM" src="https://user-images.githubusercontent.com/3868028/89822272-ed4cf680-db1d-11ea-9b2a-792a24cb5a4c.png">

**Comment by richaferry** [comment]
I think eyes-only-user-access-shortcode already closed. Is it this issue still valid @stevejburge ?

**Comment by stevejburge** [comment]
@richaferry @agapetry Yes, working on it here https://github.com/publishpress/publishpress-permissions/pull/2447

**Comment by richaferry** [comment]
Oh, ok. You should mention/link this issue in that pull request (PR) so I can easily understand it. Thanks @stevejburge 👍 
I'll take a look
